src/request.coffee

```diff
@@ -16,12 +17,23 @@ if osVersion?
 	else
 		userAgent += " (Linux; #{osVersion})"
 
+# With these settings, the device must be unable to receive a single byte
+# from the network for a continuous period of 20 minutes before we give up.
+# (reqTimeout + retryInterval) * retryCount / 1000ms / 60sec ~> minutes
+DEFAULT_REQUEST_TIMEOUT = 30000 # ms
+DEFAULT_REQUEST_RETRY_INTERVAL = 10000 # ms
+DEFAULT_REQUEST_RETRY_COUNT = 30
```

Comment:
I'm not too familiar with the supervisor codebase, but do you think it's worth adding these to the `config` object used elsewhere?